examples/java/spring-boot/README.md

```diff
@@ -0,0 +1,16 @@
+# Spring Boot application
+
+This is a Spring Boot application built from the [schemas for the federated model](https://github.com/co-cddo/federated-api-model/tree/main/schemas) in the top level of this repository.
+
+The app is deployed in a sandbox environment on PaaS, and has an [/apis endpoint](https://federated-api-model-spring-boot-sandbox.london.cloudapps.digital/apis) for returning metadata about other APIs.
+
+### Building and running locally
+To build the application locally run:
+
+`./gradlew clean build`
+
+To run the application locally run:
+
+`./gradlew bootRun`
+
+This will make the application available on port 8080.
```

Comment:
Can't request reviews of Markdown with code snippets (very well), but I'd expect more likely to have:

## Building and running locally

To build the application locally run:

```sh
./gradlew clean build
```

To run the application locally run:

```sh
./gradlew bootRun
```

This will make the application available on port 8080.